ssm/emissions.py

```diff
@@ -541,7 +555,7 @@ def initialize(self, datas, inputs=None, masks=None, tags=None):
         yhats = [self.link(np.clip(d, .1, .9)) for d in datas]
         self._initialize_with_pca(yhats, inputs=inputs, masks=masks, tags=tags)
 
-    def hessian_log_emissions_prob(self, data, input, mask, tag, x):
+    def hessian_log_emissions_prob(self, data, input, mask, tag, x, Ez):
```

Comment:
Let's raise an Exception with an error message saying that we haven't supported multiple subspaces yet.

Reply:
I replaced each assertion of single_subspace with an exception saying multiple subspaces are not supported yet.